pkg/feature/manifest.go

```diff
@@ -88,7 +88,10 @@ func (t *templateManifest) Process(data any) ([]*unstructured.Unstructured, erro
 		return nil, fmt.Errorf("failed to create file: %w", err)
 	}
 
-	tmpl, err := template.New(t.name).Funcs(template.FuncMap{"ReplaceChar": ReplaceChar}).Parse(string(content))
+	tmpl, err := template.New(t.name).
```

Comment:
Make me wonder.. could we "build time" validate the template against "test structures".... hmm